### ISSUE TITLE
[BUG] The mod doesn't work on 1.21.4.

### ISSUE THREAD

**Issue by yum13241** [Nettakrim/Souper-Secret-Settings#21]
Crash log: https://mclo.gs/uZGHsQs

Reason: Mixin error preventing the game from loading.

How to fix: Update the mod.

How to reproduce:
1. Use 1.21.4 with Fabric
2. Add the latest version at the time of writing
3. Crash

Extra stuff: Please add an option to not change the shader when drinking soup, making it controllable only with commands (for those people that like permanent color-convolve and want to use it on competitive/smp servers).

TL;DR: Update plz

**Comment by Nettakrim** [comment]
if you have a specific set of shaders you want permanently on you can make them auto apply using resourcepacks (https://github.com/Nettakrim/Souper-Secret-Settings/blob/main/ResourcepackGuide%2FPackSetup.md#entity-links)

im still in the process of updating it to latest, mojang changed a lot about how shaders work so its taking *a while*

with a bit of work you can have shaders apply with resourcepacks in vanilla, but i am aiming to add more configurability in the next update

**Comment by yum13241** [comment]
In 1.21.5, they changed the vanilla shader system again. This time though even stuff like the LSD-trip one (i forgot what it's called) is broken, because of some restrictions that weren't thought out.

**Comment by Nettakrim** [comment]
yeah 1.21.5 will take a bit to update to, i am nearly finished with 1.21.4 though

that particular change shouldnt be an issue for soup because me and dannytaylor are using a different thing for time anyway

**Comment by Nettakrim** [comment]
[:3](https://modrinth.com/mod/souper-secret-settings/version/2.0.0)
